src/main/host/descriptor/socket/inet/udp.rs

```diff
@@ -898,22 +898,11 @@ impl UdpSocket {
                 self.recv_buffer
                     .set_soft_limit_bytes(val.try_into().unwrap());
             }
-            (libc::SOL_SOCKET, libc::SO_REUSEADDR) => {
-                // TODO: implement this, tor and tgen use it
-                log::warn!("setsockopt SO_REUSEADDR not yet implemented");
-            }
-            (libc::SOL_SOCKET, libc::SO_REUSEPORT) => {
-                // TODO: implement this, tgen uses it
-                log::warn!("setsockopt SO_REUSEPORT not yet implemented");
-            }
-            (libc::SOL_SOCKET, libc::SO_KEEPALIVE) => {
-                // TODO: implement this, libevent uses it in
-                // evconnlistener_new_bind()
-                log::warn!("setsockopt SO_KEEPALIVE not yet implemented");
-            }
```

Comment:
Alternatively, keep the individual cases and make them each `warn_once_then_debug`? (And change them all to return an error)

Reply:
What would be the advantage compared to handling them like other unknown sockopts?

Reply:
We would get a warning once per unimplemented option instead of just once for the catchall. I'm fine with either way, though.

Reply:
These now log_once_then_debug and return an error (except for SO_BROADCAST).